### PR TITLE
adds testCases for runsConfig

### PR DESCRIPTION
**Pull request by Riturajcse**
https://github.com/Shippable/bat/issues/337

### CODE REVIEW

File: tests/organization-owner/github/5-projects/1.2.06.RunsConfig.GetProjectOwnerAccounts.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            projectId = nconf.get('shiptest-GITHUB_ORG_1:projectId');
+            var shippable = new Shippable(config.apiToken);
+            shippable.getProjectOwnerAccounts(projectId,
+              function (err) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                      '\n - [ ] %s Get ProjectOwnerAccounts for ' +
+                      'projectId: %s failed with error: %s' +
```
Comment (deepikasl) [inline tests/organization-owner/github/5-projects/1.2.06.RunsConfig.GetProjectOwnerAccounts.js]: use `,` here not `+`

File: tests/organization-owner/github/5-projects/1.2.15.RunsConfig.ClearProjectTimeoutWorkflow.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+              clearTimeoutMS: true
+            };
+            shippable.putProjectById(projectId, update,
+              function (err) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                      '\n - [ ] %s clearProjectTimeoutWorkflow ' +
+                      'projectId: %s failed with error: %s' +
```
Comment (deepikasl) [inline tests/organization-owner/github/5-projects/1.2.15.RunsConfig.ClearProjectTimeoutWorkflow.js]: use `,` here also

File: tests/organization-owner/github/5-projects/1.2.07.RunsConfig.SetOwnerAccountIdWorkflow.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            };
+            var shippable = new Shippable(config.apiToken);
+            shippable.putProjectById(projectId, update,
+              function (err) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                      '\n - [ ] %s SetOwnerAccountIdWorkflow for ' +
+                      'projectId: %s failed with error: %s' +
```
Comment (nranjanShippable) [inline tests/organization-owner/github/5-projects/1.2.07.RunsConfig.SetOwnerAccountIdWorkflow.js]: use `,` instead of `+`

**Comment by Riturajcse** [comment]
Closing this since need to do it in different way

**Pull request closed without merging**